fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -550,6 +558,33 @@ private void collectTableStatisticsAndCacheIcebergSplit(Table table, ScalarOpera
         scannedTables.add(key);
     }
 
+    /**
+     * To optimize the MetricsModes of the Iceberg tables, it's necessary to display the columns of the
+     * FULL or TRUNCATE type MetricsMode in the ICEBERG query profile.
+     * <br>
+     * Truncate:
+     * <p>
+     * Under this mode, value_counts, null_value_counts, nan_value_counts and truncated lower_bounds,
+     * upper_bounds are persisted.
+     * </p>
+     * Full:
+     * <p>
+     * Under this mode, value_counts, null_value_counts, nan_value_counts and full lower_bounds,
+     * upper_bounds are persisted.
+     * </p>
+     */
+    private static Map<String, MetricsModes.MetricsMode> getComplexIcebergMetricsConfig(IcebergTable table) {
+        MetricsConfig metricsConf = MetricsConfig.forTable(table.getNativeTable());
+        Map<String, MetricsModes.MetricsMode> filedToMetricsMode = Maps.newHashMap();
+        for (Types.NestedField field : table.getNativeTable().schema().columns()) {
+            MetricsModes.MetricsMode mode = metricsConf.columnMode(field.name());
+            if (mode.equals(MetricsModes.Full.get()) || mode.toString().toLowerCase().startsWith("truncate")) {
```

Comment:
The default mode is `truncate(16)`. I think we don't need to record the default mode columns

Reply:
done